### PR TITLE
JobRow: Use GObject style

### PR DESCRIPTION
**Pull request by danrabbit**
also make "update state" private

### CODE REVIEW

File: src/Widgets/JobRow.vala [hunk 1]
```diff
@@ -21,15 +21,19 @@
  */
 
 public class Printers.JobRow : Gtk.ListBoxRow {
-    public Job job { get; private set; }
-    private Printer printer;
+    public Job job { get; construct set; }
+    public Printer printer { get; construct; }
```
Comment (artemanufrij) [inline src/Widgets/JobRow.vala]: use `construct set;` like other vars

Comment (tintou) [reply src/Widgets/JobRow.vala]: I would instead propose the remove the set of the job one

Comment (danrabbit) [reply src/Widgets/JobRow.vala]: Job needs a set because it's changed in a method below

Comment (tintou) [reply src/Widgets/JobRow.vala]: so `private set` would be good for me

**Pull request merged**